Cargo.toml

```diff
@@ -9,23 +9,30 @@ documentation = "https://docs.rs/fuse-rust"
 repository = "https://github.com/Blakeinstein/fuse-rust"
 homepage = "https://github.com/Blakeinstein/fuse-rust"
 categories = ["algorithms", "text-processing", "gui"]
-keywords = ["weighted-search", "search", "bitap", "fuzzy-search", "fuzzy-matching"]
+keywords = [
+  "weighted-search",
+  "search",
+  "bitap",
+  "fuzzy-search",
+  "fuzzy-matching",
+]
 description = """
 Fuse is a super lightweight library which provides a simple way to do fuzzy searching.
 Fuse-Rust is a port of Fuse-Swift, written purely in rust.
 """
 
 [dependencies]
-crossbeam-utils = { version = "0.8.14", optional = true }
+crossbeam-utils = { version = "0.8", optional = true }
+rayon = { version = "1", optional = true }
 
 [features]
-async = [ "crossbeam-utils" ]
+default = ["rayon"]
+async = ["crossbeam-utils"]
+rayon = ["dep:rayon"]
 
 [workspace]
-members = [
-  "examples/search-bar"
-]
+members = ["examples/search-bar"]
 
 [[example]]
 name = "chunk-search"
-required-features = ["async"]
\ No newline at end of file
+required-features = ["async"]
```

Comment:
We didnt add a required-features flag for `chunk-search-rayon`?

Reply:
yeah...